cmd/test/cmd.go

```diff
@@ -111,6 +111,9 @@ func doRun(ctx context.Context, servicesToTest []string, options *Options, ioCtr
 
 	// Loads, updates and uses the context from path. If not found, it creates and uses a new context
 	if err := contextCMD.LoadContextFromPath(ctx, options.Namespace, options.K8sContext, options.ManifestPath, contextCMD.Options{Show: true}); err != nil {
+		if err.Error() == fmt.Errorf(oktetoErrors.ErrNotLogged, okteto.GetContext().Name).Error() {
```

Comment:
We should probably create a specific error and being able to use `error.Is()` instead of comparing string values of the error. For example, not sure if we could have some problem with a trailing slash (`/`) at the end of the context name at some point.

Not needed to do as part of this PR, but it should be the way to go